### DISCUSSION TITLE
what is a board

### DISCUSSION THREAD

**Discussion by 15696** [ClemBotProject/ClemBot#316] [Q&A]
what is it

**Comment by mudkipdev** [comment]
im waiting

**Comment by rootanirban** [comment]
A **board** is a general term, and its meaning depends on context:

### Common meanings:

* **Flat surface:** A thin, flat piece of material (like wood or plastic), e.g., a *whiteboard* or *blackboard*.
* **Group of people:** A governing body that makes decisions, e.g., a *board of directors*.
* **Notice/display area:** A place where information is posted, like a *notice board*.
* **Games:** The surface used to play games, e.g., a *chessboard*.
* **Technology:** A circuit platform, like a *motherboard* in a computer.

**Reply by mudkipdev** [reply]
Not fucking helpful

**Reply by rootanirban** [reply]
tell me which board you talk about

**Reply by mudkipdev** [reply]
@Jay-Madden will explain
